### PR TITLE
conplete isSanta practice - Hoang Nguyen

### PR DESCRIPTION
**Pull request by hoangnguyen9384**
Screen Shot 2020-06-22 at 9 50 07 PM

### CODE REVIEW

File: is-santa-coming/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+</head>
+<body>
+
+
+  <script>
+
+    function isSantaComing(dateObject) {
+       var month = dateObject.getMonth();
+       var date = dateObject.getDate();
+      if (month === 12 && date === 24) {
```
Comment (BrettAlbright) [inline is-santa-coming/index.html]: Hey Jay, Month needs to be 11 here as the months are zero indexed like arrays!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

File: is-santa-coming/index.html (hunk 1 above)
Comment (hoangnguyen9384) [reply is-santa-coming/index.html]: Ok i got it. Thanks Bret

**Review by Zhiwutian — COMMENTED** [review]
👍

**Pull request merged**